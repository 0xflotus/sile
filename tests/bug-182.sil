\begin[papersize=a4]{document}%
\raggedright{\glue[width=80mm]ABC}
\end{document}