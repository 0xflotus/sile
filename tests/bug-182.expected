Set paper size 	595.275597	841.8897729
Begin page
Mx 	256.5354
My 	48.5252
Set font 	Gentium Plus;10;400;;normal;;LTR
T	36 37 38	(ABC)
Mx 	295.2916
My 	780.6177
T	20	(1)
End page
Finish
